ui/app/ducks/swaps/swaps.js

```diff
@@ -88,6 +99,32 @@ const slice = createSlice({
     setToToken: (state, action) => {
       state.toToken = action.payload
     },
+    swapCustomGasModalPriceEdited: (state, action) => {
+      state.customGas = {
+        ...state.customGas,
+        price: action.payload,
+      }
+    },
+    swapCustomGasModalLimitEdited: (state, action) => {
+      state.customGas = {
+        ...state.customGas,
+        limit: action.payload,
+      }
+    },
+    swapGasPriceEstimatesFetchStarted: (state, action) => {
```

Comment:
Nit: the `action` here is redundant. We can infer what the state change should be solely from the event itself.